packages/@sanity/desk-tool/src/components/paneItem/ReferencedDocHeading.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {ReactNode} from 'react'
+import {Popover, Box, Text, Heading, Flex, Button, Stack, ThemeColorProvider} from '@sanity/ui'
+import {LinkIcon} from '@sanity/icons'
+import {TitleText, LinkCircle, HorizontalLine, StyledBox} from '../pane/PaneHeader.styles'
+import {useDeskToolSetting} from '../../settings'
+import {ReferencedDocTooltip} from './ReferencedDocTooltip'
+
+export function ReferencedDocHeading({title}: {title?: ReactNode}) {
+  const [hidePopover, setHidePopover] = useDeskToolSetting(
+    'desk-tool',
+    'referenced-doc-has-confirmed-dialog',
+    false
+  )
+
+  if (hidePopover) {
+    return (
+      <Flex>
+        <TitleText tabIndex={0} textOverflow="ellipsis" weight="semibold">
+          {title}
+        </TitleText>
+        <ReferencedDocTooltip />
+      </Flex>
+    )
+  }
+
+  const onClose = () => {
+    setHidePopover(true)
+  }
+
+  return (
+    <ThemeColorProvider tone="default">
+      <Box marginLeft={2}>
+        <Popover
+          content={<InnerPopover onClose={onClose} />}
+          placement="bottom-start"
+          referenceElement={document.getElementById('referenceTitle')}
+          portal
+          open
+        >
+          <Box display="flex">
+            <TitleText id="referenceTitle" tabIndex={0} textOverflow="ellipsis" weight="semibold">
+              <Box marginBottom={2}>{title}</Box>
+            </TitleText>
+            <ReferencedDocTooltip />
+          </Box>
+        </Popover>
+      </Box>
+    </ThemeColorProvider>
+  )
+}
+
+function InnerPopover({onClose}: {onClose?: () => void}) {
+  return (
+    <StyledBox as={'div'}>
```

Comment:
Stylistically, we prefer that static string values use the regular XML-like notation, even if it has no impact on the actual output.

```suggestion
    <StyledBox as="div">
```